ui/panels-plugin/src/plugins/time-series-chart/VisualOptionsEditor.tsx

```diff
@@ -59,12 +59,12 @@ export function VisualOptionsEditor({ value, onChange }: VisualOptionsEditorProp
         control={
           <Slider
             data-testid={VISUAL_CONFIG.line_width.testId}
-            defaultValue={DEFAULT_LINE_WIDTH}
+            value={value.line_width ?? DEFAULT_LINE_WIDTH}
             valueLabelDisplay="auto"
             step={0.5}
             marks
             min={0.5}
-            max={10}
+            max={4}
```

Comment:
Similar optional suggestion as above about pulling these out into constants. This PR is already doing a lot, so totally ok to do these in a follow-up or the future if you don't want to mess with it right now.